src/Models/T1_relaxometry/mp2rage.m

```diff
@@ -209,9 +209,9 @@
 
             [T1corrected, MP2RAGEcorr] = T1B1correctpackageTFL(data.B1map,MP2RAGEimg,[],MP2RAGE,[],invEFF);
             
-            FitResult.T1 = T1corrected.img;
```

Comment:
What did these names have to do with masking? I think changing the names might break existing pipeline because it's a change to the API